app/controllers/api/works_controller.rb

```diff
@@ -10,12 +10,8 @@ def create
     attachment_attr = work_attr.delete("attachments_attributes")
     @work = Work.new(work_attr)
     if @work.save
-      @work.attachments = attachment_attr.map do |a|
-        attachment_params = {}
-        attachment_params[:image] = a
-        attachment_params[:work_id] = @work.id
-        @work.attachments.create(attachment_params)
-      end
+      @work.images.attach(attachment_attr)
```

Comment:
Hmm...looks like this makes it so that you can only attach one image per work. Can we make this multiple images?

Reply:
i think we good here